packages/KIcon/CHANGELOG.md

```diff
@@ -3,61 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [6.1.2](https://github.com/Kong/kongponents/compare/@kongponents/kicon@6.1.1...@kongponents/kicon@6.1.2) (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
-
-
-
-## [6.1.1](https://github.com/Kong/kongponents/compare/@kongponents/kicon@6.0.1-alpha.3...@kongponents/kicon@6.1.1) (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
-
-
-
-## [6.0.1-alpha.3](https://github.com/Kong/kongponents/compare/@kongponents/kicon@6.0.1-alpha.2...@kongponents/kicon@6.0.1-alpha.3) (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
-
-
-
-## 6.0.1-alpha.2 (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
-
-
-
-## 6.0.1-alpha.1 (2021-11-16)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
-
-
-
-## 6.0.1-alpha.0 (2021-11-16)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
-
-
-
-## 5.0.9-alpha.0 (2021-11-16)
-
-**Note:** Version bump only for package @kongponents/kicon
-
-
+## [6.1.2](https://github.com/Kong/kongponents/compare/@kongponents/kicon@5.0.8...@kongponents/kicon@6.1.2) (2021-11-17)
 
+### BREAKING CHANGES
+  * changed the way the svg is rendered
```

Comment:
Add more detail, something like:

> SVGs are now wrapped with `<span>` and `KIcon`-specific classes are now applied to the wrapper.

I think pointing out the SVG is now not the tag to target here is important since so many styles changed as a result.

Reply:
Done!